modules/ROOT/pages/setting-environment-variables.adoc

```diff
@@ -0,0 +1,28 @@
+= Setting Environment Variables
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, environment variables, system variables, system properties
+
+If you're deploying multiple applications through a Shared Resources structure, then you shouldn't set any properties in the project properties files, as there might potentially be conflicts between the various apps that share a domain. Instead, you can set environment variables over the scope of the deployed app, its domain and other apps under that domain.
+
+The steps below explain how to set these environment variables in Studio. Note that these variables will only be used when deploying locally through Studio, when deploying your applications to Mule Standalone, you must provide these variables through the command line instead.
+
+. Open the dropdown menu next to the play button and select *Run Configurations*.
++
+image::run-configurations-1.png[run+configurations+1]
+
+. Select the corresponding *Run Configuration.* If one does not exist for your project, you can create one via the *Create New* icon above.
++
+image::run-configurations-4.png[run+configurations+4]
+
+. Then pick the *Environment* tab, and add the variables you need by clicking the *New* button.
++
+image::run-configurations-2.png[run+configurations+2]
+
+. The environment variables you create are defined at domain level, and will affect all of the applications associated to that domain when deploying locally via Studio.
+
+== See Also
+
+* Learn more about xref:shared-resources.adoc[Shared Resources]
+* If you're deploying a single application, you can still set up xref:configuring-properties.adoc[Application Properties]
```

Comment:
We don't put text like this in the See Also section afaik. This is more of a note somewhere in the main body of the topic.